src/client/callers/agentUnlock.ts

```diff
@@ -0,0 +1,10 @@
+import { HandlerTypes, UnaryCaller } from "@matrixai/rpc";
+import {ClientRPCRequestParams, ClientRPCResponseResult} from "../../client/types";
```

Comment:
we're in the client directory, the path should be `../types`

Reply:
Actually, this should be removed. we don't use these types directly anymore. Linter should've complained about that so you still need to lint.